experimental/Schemes/src/ToricBlowups/types.jl

```diff
@@ -11,6 +11,28 @@
   center::Union{ToricIdealSheafFromCoxRingIdeal, IdealSheaf}
   exceptional_divisor::ToricDivisor
 
+  # This constructor handles the case in which the new ray is already part of the rays of the toric variety.
+  function ToricBlowdownMorphism(v::NormalToricVariety, new_ray::AbstractVector{<:IntegerUnion})
+
+    # Compute position of new ray and verify that it is already among the rays of the given variety v
+    rs = matrix(ZZ, rays(v))
+    position_new_ray = nothing
+    for i in 1:nrows(rs)
+      if new_ray == rs[i, :]
+        position_new_ray = i
+        break
+      end
+    end
```

Comment:
```suggestion
    position_new_ray = findfirst(i->new_ray==rs[i, :], 1:nrows(rs))
```
Wouldn't this work as well? Also one might consider putting this behind some `@check` flag?

Reply:
Will try this. Thank you for the suggestion.

Reply:
Just included this in the code.